couchbase-database-plugin/main.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
This should be in `cmd/vault-plugin-database-couchbase/main.go`

Reply:
Done.